src/app/code/community/Zendesk/Zendesk/sql/zendesk_setup/mysql4-install-1.4.0.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright 2015 Zendesk
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$installer = $this;
+$installer->startSetup();
+$installer->run("CREATE TABLE IF NOT EXISTS `{$installer->getTable('zendesk_settings')}` (
```

Comment:
Are these tables still required as this PR no longer includes the individual user settings?